modules/creating-a-project-workbench.adoc

```diff
@@ -55,6 +55,23 @@ endif::[]
 .. Configure the storage for your {productname-short} cluster.
 ... Select *Create new persistent storage* to create storage that is retained after you log out of {productname-short}. Complete the relevant fields to define the storage.
 ... Select *Use existing persistent storage* to reuse existing storage and select the storage from the *Persistent storage* list.
+.. To use a *Data connection*, select the *Use a data connection* check box.
```

Comment:
Bolding a noun (in this case **Data connection**) to also represent the name of a label or section in the UI goes against our style guidelines. Also, IBM style advises "checkbox" (see https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#checkbox).

Therefore, I suggest something like the following: 

`To use a data connection, in the *Data connections* section, select the *Use a data connection* checkbox.`

(I know that there's repetition in that, but I think it's the appropriate formulation).

Reply:
Thanks @jbyrne-redhat - I just addressed all the comments in the latest commit, please review.

Reply:
![Screenshot 2023-11-17 at 9 45 27 AM](https://github.com/opendatahub-io/opendatahub-documentation/assets/149716269/3179af0f-740c-4117-bc68-baa8c3689a07)